CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.25.0] - 2023-07-30
```

Comment:
don't add this line yet, we are not releasing this version yet. You can add your entry under `[Unreleased]` section (just simply drop this line).

Reply:
yeap, I got it. I just updated it.